runtime/shared_common/OSCachesysv.cpp

```diff
@@ -1745,23 +1745,23 @@ SH_OSCachesysv::getCacheStatsHelper(J9JavaVM* vm, const char* cacheDirName, UDAT
 			return -1;
 		}
 #endif /* defined(J9ZOS390) */
-		cacheInfo->os_shmid = (statbuf.shmid!=(UDATA)-1)?statbuf.shmid:(UDATA)J9SH_OSCACHE_UNKNOWN;
+		cacheInfo->os_shmid = statbuf.shmid;
 		/* os_semid is populated in getJavacoreData() as we can't access _semid here */
-		cacheInfo->os_semid = (UDATA)J9SH_OSCACHE_UNKNOWN;
-		cacheInfo->lastattach = (statbuf.lastAttachTime!=-1)?(statbuf.lastAttachTime*1000):J9SH_OSCACHE_UNKNOWN;
-		cacheInfo->lastdetach = (statbuf.lastDetachTime!=-1)?(statbuf.lastDetachTime*1000):J9SH_OSCACHE_UNKNOWN;
-		cacheInfo->createtime = J9SH_OSCACHE_UNKNOWN;
-		cacheInfo->nattach = (statbuf.nattach!=(UDATA)-1)?statbuf.nattach:(UDATA)J9SH_OSCACHE_UNKNOWN;
+		cacheInfo->os_semid = J9SH_OSCACHE_UNKNOWN;
+		cacheInfo->lastattach = (statbuf.lastAttachTime != -1) ? (statbuf.lastAttachTime * 1000) : (IDATA)J9SH_OSCACHE_UNKNOWN;
+		cacheInfo->lastdetach = (statbuf.lastDetachTime != -1) ? (statbuf.lastDetachTime * 1000) : (IDATA)J9SH_OSCACHE_UNKNOWN;
```

Comment:
R-values on the left, please. e.g.  `(-1 != statbuf.lastAttachTime)`.

Reply:
Oops, updated.